client/extensions/woocommerce/state/sites/product-categories/selectors.js

```diff
@@ -1,7 +1,7 @@
 /**
  * External dependencies
  */
-import { get, isArray, omit, some, range, values } from 'lodash';
+import { get, omit, some, range, values } from 'lodash';
```

Comment:
Not sure if this isn't already done in another PR, but we could probably also use native [`some`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) as well?

Reply:
Good call, but yep, that's for another PR 😉  We're addressing those one by one, step by step.